src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaFigure, Drawable {
+    protected String color;
+    protected String name;
+    protected double area;
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    @Override
+    public void area() {
+
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: " + name + " area: " + area);
+    }
```

Comment:
remove implementation from here and add this info inside of every figure draw() method